src/components/@shared/VerifiedBadge/index.tsx

```diff
@@ -22,11 +25,25 @@ export function Badge({
       className={cx({
         mainLabel: true,
         isValid,
+        isWarning: matchVerifiable === false,
         [className]: className
       })}
     >
+      {matchVerifiable === false && (
+        <Tooltip content="gx:dependsOn could not be found" />
+      )}
```

Comment:
suggestion(blocking): it is possible to wrap items in a tooltip component for a bit better UX

right now, only hovering the little icon pops up the tooltip, but it would be useful if the tooltip always shows when hovering the text. This can be part of the component overall, allowing to pass an additional string to display as tooltip


![image](https://github.com/user-attachments/assets/1a220c6c-507f-40e9-ac5b-b56c407f8493)